### PR TITLE
Bug/1885 get attr value date

### LINKED ISSUE #1885: [BUG] in GET /v2/entities/<id>/attrs/<attr_name>/value with date as attr_type, no returns correctly
in GET `/v2/entities/<id>/attrs/<attr_name>/value` with date as attr_type, doesn't return a correct ISO8601 formated string.

### Create request
```
POST http://localhost:1026/v2/entities
Fiware-Service: happy_path
Fiware-ServicePath: /test
Content-Type: application/json
{"id": "room_6", "timestamp": {"type": "date", "value": "2035-06-15T07:21:24.238Z"}}
```

### Get entity request
```
GET http://localhost:1026/v2/entities/room_6
Fiware-Service: happy_path
Fiware-ServicePath: /test
```

##### Get entity response
```
{"id":"room_6","type":"none","timestamp":{"type":"date","value":"2035-06-15T07:21:24.00Z","metadata":{}}}
```

### Get attribute value request
```
GET http://localhost:1026/v2/entities/room_6/attrs/timestamp/value
Fiware-Service: happy_path
Fiware-ServicePath: /test
Accept: text/plain
```

##### Get attribute value response
```
2065504884.000000
```

### PR DESCRIPTION
**Pull request by crbrox**
Fixes #1885 

@kzangeli 
@fgalan

### CODE REVIEW

File: src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp [hunk 1]
```diff
@@ -136,7 +140,16 @@ std::string getEntityAttributeValue
...
+
+        if (attributeType == DATE_TYPE)
+        {
+          answer = isodate2str(attribute.pcontextAttribute->numberValue);
+        }
+        else
+        {
+          answer =  attribute.pcontextAttribute->getValue();
+        }
+        TIMED_RENDER(answer = answer);
```
Comment (kzangeli) [inline src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp]: We just got really fast here ... :-)
I imagine the TIMED_RENDER macro should be used for the two assignments of answer.

**Comment by kzangeli** [comment]
LGTM

File: src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp (hunk 1 above)
Comment (crbrox) [inline src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp]: glups ....
Fixed at b31e920

**Comment by fgalan** [comment]
LGTM

**Pull request merged**